front_end/ndb/module.json

```diff
@@ -18,14 +18,6 @@
             "settingType": "string",
             "defaultValue": ""
         },
-        {
-            "type": "setting",
-            "category": "Debugger",
-            "title": "Wait at end",
-            "settingName": "waitAtEnd",
```

Comment:
This is now managed via https://github.com/ChromeDevTools/devtools-frontend/commit/24ec659c09459167a3e59044a6b4654ad40ba1fa ? 
if so, don't you need a roll?

Reply:
yep, I am waiting for roll, thank you!